### ISSUE TITLE
New Search Page Problems - not returning results

### ISSUE THREAD

**Issue by ewommack** [ArctosDB/arctos#5420]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

From Arctos contact email: 

Acord, Brian@Wildlife <Brian.Acord@wildlife.ca.gov>
5:25 PM (3 hours ago)
to arctoscontact@googlegroups.com

Good afternoon,

The new form doesn’t work.

 

This is in response to the request for feedback on the new Specimen Search form (http://arctos.database.museum/search.cfm) that will be replacing the old form (http://arctos.database.museum/SpecimenSearch.cfm). I query for museum data weekly. I’ve attempted to use the new form at least 4 or 5 times now over the past week and half (in current/updated versions of FireFox and Chrome). Not once have I been able to get it to do anything other than show me blue dots flashing across the bottom of the web page. It does this when I initially open the webpage and continues when I add custom search fields and, especially disappointing, when I run a search. In other words, I never get a result. I easily do the same query in the “old” search form and it’s fine, no problem.

 

Further feedback, the webpage request for feedback goes to GitHub where you are asked to sign in or create an account. This is poor. I want to give you feedback, but I have no interest in creating an account with GitHub. I suspect this is preventing further feedback about the site.

 

Finally, I understand change can be annoying, so this is probably chalked-up to that. Anyhow, the layout of the new form and trying to figure out what the fields are and where they are was a bit annoying. I’m sure if it worked and I used it more I’d get used to it.

 

Sincerely,

Brian

 

--

Brian Acord // CNDDB zoology lead

[CDFW](https://www.wildlife.ca.gov/)//[Biogeographic Data Branch](https://www.wildlife.ca.gov/Explore/Organization)

https://www.wildlife.ca.gov/Data/CNDDB

 

New mailing address:

California Department of Fish & Wildlife (CDFW)

California Natural Diversity Database (CNDDB)

PO Box 944209

Sacramento, CA 94244

 

New physical address:

1700 9th Street, 4th floor, Sacramento

**Comment by Jegelewicz** [comment]
> Hello Brian,
> 
> Thank you for taking the time to let us know how the new search page is going. We have to update it, as we've reached a point where we have out grown the previous system. We're trying to find something that will work for everyone, with understanding that many of us are very used to, and good with the old design. Having feedback and information from all users is really helpful, and we appreciate you taking the time to send it to us.
> 
> I am going to create a Github issue for your comments, so we can track them in our system, but please feel free to continue to respond to this email if Github is too difficult for you to use. The issue is here: [https://github.com/ArctosDB/arctos/issues/5420](https://gcc02.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2FArctosDB%2Farctos%2Fissues%2F5420&data=05%7C01%7CBrian.Acord%40wildlife.ca.gov%7C4518771f720d4969e73108dae88a08c8%7C4b633c25efbf40069f1507442ba7aa0b%7C0%7C0%7C638077976818787609%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=hQcLbpIEW%2F9i9iG5c1SgSV89I3EF65hiWm4f%2B%2FcdIvc%3D&reserved=0)
> 
> Can you please elaborate on the specific steps you take when you have been attempting searches and the new page has not returned results? Screen shots, and specific steps (e.g. search terms, systems of sorting windows, etc.) help us understand where and how things may not be working.
> 
> Thank you,

Beth Wommack

**Comment by Jegelewicz** [comment]
> Good morning,
> 
> As mentioned, the page doesn’t do anything. From the time I open the page, or if I try and enter anything, all that happens is blue dots spin around near the bottom. If I enter any information to query, it does nothing different, i.e. nothing at all, when I hit Submit Query.
> 
> Sincerely,

Brian

**Comment by ewommack** [comment]
A set of screen shots that he included:
![20221228 arctos CTS Storer Capture](https://user-images.githubusercontent.com/18562884/210030173-969a4e47-9e90-4d1b-a7c5-3ec12db4101f.jpeg)
![20221228 arctos Capture](https://user-images.githubusercontent.com/18562884/210030178-c7a55e3d-f353-47e3-918f-713cb0801028.jpeg)
![20221228 arctos CTS Capture](https://user-images.githubusercontent.com/18562884/210030179-33a457a4-5230-4629-bf49-8fce5f9d7927.jpeg)

**Comment by ewommack** [comment]
Hello Brian,

Thank you for the screen shots. I tried your first search on my Mac laptop (running Chrome) and it took about half a minute to pull up 16,492 entries. So it wasn't instantaneous, but it did return results. I tried your second query and got 5,511 results with no blue dots at all. So it looks like the fields are working, but there may be a bug you have discovered.

When I open the page it first opens all of the search fields for me, but I do not get any blue dots. If you clear your cache and reopen the program can you try accessing the page again for us and seeing if the blue dots appear before you enter any fields please?

And while we are trying to figure out things, please feel free to use the legacy form. We still have it available through the link at the top of the search page.

Thank you,
Beth

**Comment by Jegelewicz** [comment]
> Good morning,
> 
> As mentioned, the page doesn’t do anything. From the time I open the page, or if I try and enter anything, all that happens is blue dots spin around near the bottom. If I enter any information to query, it does nothing different, i.e. nothing at all, when I hit Submit Query.
> 
> Sincerely,

Brian

**Comment by Jegelewicz** [comment]
> HI Brian, thanks for the detailed feedback. (sorry I was on an email break last week so wasnt able to respond to your earlier emails). Obviously we are transitioning to a new method of handling queries both on the frontend and backend. Like Beth, I wasnt able to replicate all of the behaviours and delays that you note, so could I ask a favor? Perhaps a quick call when convenient for you would help us. Meanwhile I will get your feedback on our list to troubleshoot or at least figure out how to make less annoying. BTW, the URL with search parameters is one of the new functions on the search page so thanks for sending those links. Those did work for me! Thanks again and let me know if you are up for a follow-up call.
> 
> Happy New Year's!

Best, Michelle

**Comment by Jegelewicz** [comment]
> Good afternoon,
> 
> I have a feeling it’s not you, it’s me.  ;)
> 
> OK, not me per se, but my state government network. I forwarded my email below with all the links to my personal email account and opened them all without issue on my personal laptop. And then I tried again on my work computer and some of the URLs still loaded with the blue dots. It looks like I’m having better luck with the shorter URLs. If no one else is reporting issues, then it’s probably best to assume it’s our network/firewall. If I keep running into issues I’ll start a help desk ticket with our IT guys.
> 
> Sincerely,
> 

Brian

**Comment by dustymc** [comment]
I think Brian just got caught up in some instability (AWG copied on an email), but I'm happy to dig deeper if he can re-create this from work or whatever. @mkoo feel free to pass on my email or phone.

Github is awesome because we don't get spam and things don't get lost, but if someone's willing to file issues from email (or whatever) we might add alternate contacts to https://github.com/ArctosDB/internal/issues/211
